src/main/java/seedu/flashnotes/logic/commands/ClearCommand.java

```diff
@@ -17,7 +17,7 @@
     @Override
```

Comment:
Perhaps the "Address book" in MESSAGE_SUCCESS should be changed to "Flash notes"?

Reply:
Done, thanks for the catch!